src/fusor/examples/bcr_abl1.json

```diff
@@ -89,22 +66,13 @@
       "_id": "interpro:IPR000980",
       "label": "SH2 domain",
       "sequence_location": {
-        "id": "fusor.location_descriptor:NP_005148.2",
-        "type": "LocationDescriptor",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NP_005148.2",
```

Comment:
can we retain this identifier as a mapping, or something of that nature?

Reply:
I think so? I was more following the convention in `element_genomic_end` and `element_genomic_start` where I removed the identifier